notebooks/vertex_genai/solutions/vertex_llm_tuning_with_gemini.ipynb

```diff
@@ -0,0 +1,1424 @@
+{
```

Comment:
<p>Add a line here mention that the tuning takes time so folks are not surprised if it takes multiple hours. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/507/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
You can mention that for epochs = 1 and adapter size = 1 it should take 1 hour to finish fine tuning.

Reply:
<p>Great! Thanks! Done!</p>